ingest-app/src/cmr/ingest/services/granule_bulk_update/size/echo10.clj

```diff
@@ -0,0 +1,143 @@
+(ns cmr.ingest.services.granule-bulk-update.size.echo10
+  "Contains functions to update ECHO10 granule xml for size bulk granule update."
+  (:require
+   [clojure.data.xml :as xml]
+   [clojure.string :as string]
+   [clojure.zip :as zip]
+   [cmr.common.services.errors :as errors]
+   [cmr.common.xml :as cx]))
+
+(def ^:private tags-after-data-granule
+  "Defines the element tags that come after DataGranule in ECHO10 Granule xml schema"
+  #{:PGEVersionClass :Temporal :Spatial :OrbitCalculatedSpatialDomains :MeasuredParameters
+    :Platforms :Campaigns :AdditionalAttributes :InputGranules :TwoDCoordinateSystem :Price
+    :OnlineAccessURLs :OnlineResources :Orderable :DataFormat :Visible :CloudCover
+    :MetadataStandardName :MetadataStandardVersion :AssociatedBrowseImages :AssociatedBrowseImageUrls})
+
+(def ^:private tags-after-size-in-bytes
+  "Defines the element tags that come after size in a DataGranule element"
+  #{:SizeMBDataGranule :Checksum :ReprocessingPlanned :ReprocessingActual :ProducerGranuleId :DayNightFlag :ProductionDateTime
+    :LocalVersionId :AdditionalFile})
+
+(def ^:private tags-after-size-in-mb
+  "Defines the element tags that come after size in a DataGranule element"
+  #{:Checksum :ReprocessingPlanned :ReprocessingActual :ProducerGranuleId :DayNightFlag :ProductionDateTime
+    :LocalVersionId :AdditionalFile})
```

Comment:
These could probably be coalesced, but I couldn't come up with a clean way to accomplish it, so I left them as-is.